app/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersPartnershipCompanyRegistrationNumberController.scala

```diff
@@ -34,9 +34,9 @@ class BusinessPartnersPartnershipCompanyRegistrationNumberController @Inject()(
   import actions._
 
   val businessType: String = "limited-liability-partnership"
-  val companyName = "Test Partnership"
-  val backLink = "#"
-  val postAction = routes.BusinessPartnersPartnershipCompanyRegistrationNumberController.next()
+  val companyName: String = "Test Partnership"
+  val backLink: String = routes.BusinessPartnerPartnershipTradingNameController.load().url
+  val postAction: Call = routes.BusinessPartnersPartnershipCompanyRegistrationNumberController.next()
```

Comment:
Post route is incorrect: 

Then I shall be taken to the "Does the partnership have a UK VAT registration number?" page 
**/fhdds/business-partners/partnership-vat-registration-number**